### ISSUE TITLE
about InternVL−Chat−V1.5 8 bit quantization

### ISSUE THREAD

**Issue by tairen99** [InternLM/lmdeploy#1727]
Hi all,

Thank you for your good works!

I noticed the InternVL−Chat−V1.5-Int8 inference is slow as mentioned in [link](https://github.com/OpenGVLab/InternVL/issues/157).

Do you support using [link](https://github.com/InternLM/lmdeploy/blob/main/docs/en/quantization/w8a8.md) to quantize it? so it may improve the inference speed?

If you support it, what I need to prepare and follow? new to the quantization.

Thank you again!

**Comment by lvhan028** [comment]
Hi, @tairen99 
Thanks for your support to LMDeploy. 

`InternVL−Chat−V1.5-Int8` is a model that has been processed with bnb quantization, which can be executed by `bitsandbytes`. It helps decrease the GPU mem occupation. However, the inference speed is not impressive. 

LMDeploy hasn't supported bnb quantization and we have plans to do that yet.

LMDeploy provides `W8A8` quantization for LLMs rather than VLMs. The W8A8-quantized LLM model can be inferred by the PytorchEngine.

To sum up, LMDeploy doesn't currently support W8A8 quantization and inference for VLMs, including the InternVL series

**Comment by tairen99** [comment]
> Hi, @tairen99 Thanks for your support to LMDeploy.
> 
> `InternVL−Chat−V1.5-Int8` is a model that has been processed with bnb quantization, which can be executed by `bitsandbytes`. It helps decrease the GPU mem occupation. However, the inference speed is not impressive.
> 
> LMDeploy hasn't supported bnb quantization and we have plans to do that yet.
> 
> LMDeploy provides `W8A8` quantization for LLMs rather than VLMs. The W8A8-quantized LLM model can be inferred by the PytorchEngine.
> 
> To sum up, LMDeploy doesn't currently support W8A8 quantization and inference for VLMs, including the InternVL series

Hi @lvhan028,

Thank you for your clarification.
